modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -2410,9 +2325,13 @@ private ObjectEntry _getObjectEntry(
 			objectDefinition, serviceBuilderObjectEntry);
 		_checkRootDescendantNode(serviceBuilderObjectEntry, false);
 
+		com.liferay.object.model.ObjectEntry serviceBuilderParentObjectEntry =
+			_objectEntryService.fetchObjectEntry(
+				serviceBuilderObjectEntry.getRootObjectEntryId());
```

Comment:
We should pass the direct parent object entry instead of the root object entry.
We should only pass it if this entry is in the root model context, right?